### PR TITLE
feat(emoji-key): added Project Manager

### LINKED ISSUE #141: Project Manager
I sugest 📋 (Clipboard)

### PR DESCRIPTION
**Pull request by Berkmann18**
Resolves #141.

### CODE REVIEW

**Comment by Berkmann18** [comment]
Yes, indeed, do you think it would be necessary to clarify that in the emoji-key?

File: docs/emoji-key.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Emoji/Type | Represents | Comments
 🚧 <br /> `maintenance` | Maintenance | people who help in maintaining the repo, links to commits by the user on this project
 📦 <br /> `platform` | Packaging | porting to support a new platform |
 🔌 <br /> `plugin` | Plugin/utility libraries | links to the repo home
+📆 <br/> `pm` | Project Management | |
```
Comment (jakebolam) [inline docs/emoji-key.md]: Could we use the full word?

Comment (Berkmann18) [reply docs/emoji-key.md]: Sure.

File: docs/emoji-key.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ Emoji/Type | Represents | Comments
 🚧 <br /> `maintenance` | Maintenance | people who help in maintaining the repo, links to commits by the user on this project
 📦 <br /> `platform` | Packaging | porting to support a new platform |
 🔌 <br /> `plugin` | Plugin/utility libraries | links to the repo home
-📆 <br/> `pm` | Project Management | |
+📆 <br/> `projectManagement` | Project Management | |
```
Comment (Berkmann18) [inline docs/emoji-key.md]: @jakebolam Is this okay?

**Comment by MatheusRV** [comment]
Could you merge this?

**Comment by Berkmann18** [comment]
@MatheusRV I could, just waiting on @jakebolam to see if he's okay with the latest changes.

**Comment by jakebolam** [comment]
LGTM

**Pull request merged**